src/styles/navbar.scss

```diff
@@ -2,143 +2,196 @@
-  display: block;
+  display: flex;
+  flex: 1 0 auto; // flex: grow shrink basis;
+  align-items: center;
 }
-
 .navRightItems ul {
   display: flex;
-  font-size: 17px;
+  list-style: none;
+  font-size: 1.05rem;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  flex: 1 0 100%;
 }
 
+/* Since it's flex margins not needed, and specified indivudually when needed */
 .navRightItems li {
   display: block;
   margin: 0;
-  padding: 0 0 0 0.3em;
+  padding: 0;
   line-height: 1.6em;
+  font-size: 1.05rem;
 }
 
 .navRightItems li a {
   display: block;
   position: relative;
 }
+.navRightItems li a .nav-link {
+  display: inline;
+}
 
 .hnItem.contactButton {
   display: none;
 }
 
-/* Styling Get started button */
+/* to push menu left from logo*/
+.navRightItems li:first-child{
+  margin-left: auto; 
+  margin-right: 1.7rem;
+}
+.navRightItems li:first-child > a{
+  margin-left: 10px; //extra margin to compensate for down caret absent at "help" 
+}
+.navRightItems li:nth-child(2),
+.navRightItems li:nth-child(3) {
+  margin: 0 1.7rem;
+}
+
+/* Styling Get started button, margin is needed to occupy free space */
 .navRightItems li:nth-child(4) {
   margin-left: auto;
-  -webkit-transition: box-shadow .25s ease, -webkit-transform .25s ease;
-  transition: box-shadow .25s ease, -webkit-transform .25s ease;
-  transition: box-shadow .25s ease, transform .25s ease;
-  transition: box-shadow .25s ease, transform .25s ease, -webkit-transform .25s ease;
 }
 
+/* Button styling: extra margin here is unnecessary since it's taken care of by parent element */
 .navRightItems li:nth-child(4) > a {
   background: #38c690;
   border: 1px solid transparent;
+  background-color: #38c690;
+  border-color: #38c690;
   color: #fff;
-  padding: 9px 30px;
-  box-shadow: 0 0.29rem 0.58rem rgba(0, 0, 0, .15);
-  border-radius: 9.6px;
+  padding: 10px 30px;
+  margin: 0;
+  font-size: 1.05rem;
+  box-shadow: 0 .5rem 1rem rgba(0,0,0,.15);
+  border-radius: .6rem;
   white-space: nowrap;
   text-decoration: none;
-  -webkit-transition: box-shadow .25s ease, -webkit-transform .25s ease;
-  transition: box-shadow .25s ease, -webkit-transform .25s ease;
-  transition: box-shadow .25s ease, transform .25s ease;
-  transition: box-shadow .25s ease, transform .25s ease, -webkit-transform .25s ease;
-}
-
-.navRightItems li:nth-child(4):hover {
-  -webkit-transform: translate3d(0, -3px, 0);
-  transform: translate3d(0, -3px, 0);
+  -webkit-transition: all .25s ease, -webkit-transform .25s ease;
+  transition: all .25s ease, -webkit-transform .25s ease;
+  transition: all .25s ease, transform .25s ease;
+  transition: all .25s ease, transform .25s ease, -webkit-transform .25s ease;
 }
-
-.navRightItems li:nth-child(4) > a:hover {
+/* Emulate lift class, aka make it as intended */
+.navRightItems li:nth-child(4) > a:hover,
+.navRightItems li:nth-child(4) > a:focus {
   background: #30a87a;
-  box-shadow: none;
+  transform: translateY(-3px);
```

Comment:
This makes the get started button "jump" instead of work the same as the large navbar. Can we make this the same as the lift class?

Reply:
Yep, added missed transitions, both desktop and mobile menu